Automattic/wp-calypso
# Color Schemes: Activate four new color schemes for Calypso dashboard

#### Changes proposed in this Pull Request

* Removes feature flags for wpcalypso and dev environments to activate four new color schemes in Calypso

#### Testing instructions

* Switch to this PR, navigate to `/me/account/` and check the Color Schemes setting.
* You should see four new color schemes, Ocean, Midnight, Contrast, and Sunset. Activating any one should change the colors of the dashboard accordingly, with no errors.
